docs/theory/UseCases.md

```diff
@@ -42,3 +40,55 @@ the first wrap key that is in turned encrypted with the second wrap key. The
 first wrap key is stored on a removable token device, e.g. YubiKey, and the
 second wrap key is sealed in a TPM NVRAM slot. For a system to boot it must
 have launched with the correct firmware and the token must be present.
+
+## Cross-links to publications mentioning TrenchBoot
+
+### Blog posts
```

Comment:
Given that all currently listed posts are "internal" to the project, perhaps it would be better to list other types of publications before blog posts. They may be seen as more valuable to some.

Reply:
https://github.com/TrenchBoot/TrenchBoot.github.io/pull/43/commits/b26596a8a53cedb97dd3c484bd77602944f42fcb#diff-069a5c1bf0e9890db784d49d99a7419ad3c39859efd0d4d305e15ee9f8a6e1feR1-R19